org.jacoco.core.test.validation.groovy/pom.xml

```diff
@@ -0,0 +1,89 @@
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.jacoco</groupId>
+        <artifactId>org.jacoco.core.test.validation</artifactId>
+        <version>0.8.3-SNAPSHOT</version>
+        <relativePath>../org.jacoco.core.test.validation</relativePath>
+    </parent>
+
+    <artifactId>org.jacoco.core.test.validation.groovy</artifactId>
+
+    <name>JaCoCo :: Test :: Core :: Validation Groovy 2</name>
+
+    <properties>
+        <gmaven.version>1.6</gmaven.version>
+        <groovy.version>2.5.3</groovy.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>org.jacoco.core.test</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.codehaus.groovy</groupId>
+            <artifactId>groovy-all</artifactId>
+            <version>${groovy.version}</version>
+            <type>pom</type>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.22.0</version>
+                <configuration>
+                    <testSourceDirectory>${project.build.sourceDirectory}</testSourceDirectory>
+                    <testClassesDirectory>${project.build.outputDirectory}</testClassesDirectory>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.codehaus.gmavenplus</groupId>
+                <artifactId>gmavenplus-plugin</artifactId>
+                <version>${gmaven.version}</version>
+                <executions>
+                    <execution>
+                        <id>compile</id>
+                        <phase>process-sources</phase>
+                        <goals>
+                            <goal>compile</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <!--
+                    Force target bytecode to 1.7 for now
+                    as Groovy does not understand values such as 8 or 9
+                    -->
+                    <targetBytecode>1.7</targetBytecode>
+                    <sources>
+                        <source>
+                            <directory>${project.basedir}/src</directory>
+                            <includes>
+                                <include>**/*.groovy</include>
+                            </includes>
+                        </source>
+                    </sources>
+                    <sourceEncoding>${project.build.sourceEncoding}</sourceEncoding>
```

Comment:
Remove - `${project.build.sourceEncoding}` is default in `gmavenplus-plugin`.

Reply:
OK